src/integrals/integrals.cc

```diff
@@ -61,20 +62,26 @@ namespace forte {
 
 #ifdef _OPENMP
 #include <omp.h>
-bool ForteIntegrals::have_omp_ = true;
 #else
 #define omp_get_max_threads() 1
 #define omp_get_thread_num() 0
-bool ForteIntegrals::have_omp_ = false;
 #endif
 
+std::map<IntegralType, std::string> int_type_label{{Conventional, "Conventional"},
+                                                   {DF, "Density fitting"},
+                                                   {Cholesky, "Cholesky decomposition"},
+                                                   {DiskDF, "Disk-based density fitting"},
+                                                   {DistDF, "Distributed density fitting"},
+                                                   {Own, "Own"},
+                                                   {Custom, "Custom"}};
+
 void set_INT_options(ForteOptions& foptions) {
     /*- The algorithm used to screen the determinant
      *  - CONVENTIONAL Conventional two-electron integrals
      *  - DF Density fitted two-electron integrals
      *  - CHOLESKY Cholesky decomposed two-electron integrals -*/
     foptions.add_str("INT_TYPE", "CONVENTIONAL",
-                     {"CONVENTIONAL", "DF", "CHOLESKY", "DISKDF", "DISTDF", "ALL", "OWNINTEGRALS"},
+    {"CONVENTIONAL", "DF", "CHOLESKY", "DISKDF", "DISTDF", "OWNINTEGRALS", "CUSTOM"},
```

Comment:
What is the difference between OWNINTEGRALS and CUSTOM?

Reply:
OwnIntegrals completely skips the generation of two-electron integrals. I am not sure if they are ever used in practice. Custom, which is currently not working and probably should be called FCIDUMP or something similar, is for reading a set of integrals from disk, potentially from other programs. Both need some work, and so it might make sense to remove these options from here. Does any of you ever use OwnIntegrals?